alarms/Boxcar/boxcar_alarm.py

```diff
@@ -17,7 +17,7 @@ class Boxcar_Alarm(Alarm):
 	_defaults = {
 		'pokemon':{
 			'title':"A wild <pkmn> has appeared!",
-			'long_message':"Available <a href='<gmaps>'>here</a> until <24h_time> (<time_left>).",
```

Comment:
I don't think it's any more optional than the time_left or the 24h_time one. Even less so, because those two are redundant. Do you really think it'll be a typical situation, that users don't want to know, that a pokemon will return after some time and how long? I don't think so, the realistic situation would be for them to never realize this feature exists at all. And for these reasons I think it has to be in the standard values.